### PR TITLE
Add mobile compatibility to Retags

### PR DESCRIPTION
**Pull request by dlmarquis**

### CODE REVIEW

**Comment by dlmarquis** [comment]
What the fuck Travis

File: Extensions/retags.js [hunk 1]
```diff
@@ -25,17 +25,29 @@ XKit.extensions.retags = {
...
+			XKit.tools.add_css('label.retags .binary_switch_label {font-size:15px; color:white; padding-bottom:15px; }','retags_mobile_label');
+		} else {
+			this.html_toggle.appendTo('.ui_notes_switcher .part-toggle');
+		}
 		$('#retags-toggle').change(function(){
 			if ($(this).prop('checked')) {
 				localStorage.setItem(toggle,'true');
 				XKit.extensions.retags.css_toggle.appendTo('head');
+				if (XKit.browser().mobile) {
+					XKit.extensions.retags.mobile_toggle.appendTo('head');
```
Comment (nightpool) [inline Extensions/retags.js]: why do this instead of using add_css?

Comment (dlmarquis) [inline Extensions/retags.js]: Keeping everything uniform. css_toggle does the same thing, and this is just the mobile addition to it.

**Comment by hobinjk** [comment]
Tested locally and works.
@homu r+

**Pull request merged**